### PR TITLE
Template Generation Navigation, Handling Account Retrieval Error, Generating status message

### PR DESCRIPTION
**Pull request by aaclan-ebi**
ebi-ait/hca-ebi-dev-team#222 - Do not redirect the user to registration page if UI cannot connect to Ingest API

ebi-ait/hca-ebi-dev-team#232 - Template generation navigation 

ebi-ait/hca-ebi-dev-team#245 - Display status while generating spreadsheet

### CODE REVIEW

**Comment by aaclan-ebi** [comment]
Sorry to mix both changes in one PR. It's a bit difficult to isolate the changes since i worked on the handling of account retrieval error first. All unit tests are now passing in this branch.

**Comment by aaclan-ebi** [comment]
I added more changes -> Display status while generating spreadsheet.

File: client/src/app/global-header/global-header.component.ts [hunk 1]
```diff
@@ -11,21 +15,31 @@ import {AaiService} from '../aai/aai.service';
 export class GlobalHeaderComponent implements OnInit {
   isLoggedIn: boolean;
   userInfo: Profile;
+  account$: Observable<Account>;
```
[the remark this answers is not in the document]
Comment (rolando-ebi) [reply client/src/app/global-header/global-header.component.ts:18]: I wouldn't change this for now, but we should consider utilising the type safety offered by typescript, especially as this is a quickly growing codebase

**Comment by aaclan-ebi** [comment]
I've applied the PR comments. Ami has tested the flow and have no comments related to this. I'm merging the changes to master now to merge changes (which is on top of Rodrey's branch feature/template-questionnaire) and test against the latest changes.

**Pull request merged**